src/types/PerformanceTable.cpp

```diff
@@ -92,13 +98,18 @@ std::vector<Perf> PerformanceTable::operator[](std::string name) const {
 void PerformanceTable::generateRandomPerfValues(unsigned long int seed,
                                                 int lower_bound,
                                                 int upper_bound) {
+  int i = 0;
   if (lower_bound > upper_bound) {
     throw std::invalid_argument(
         "Lower bound must be lower than the upper bound.");
   }
+
+  std::mt19937 gen(seed);
```

Comment:
If that's the way of getting a real uniform distribution, might be better to change getRandomUniformFloat instead !

Reply:
done